package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "atxfloodsadmin",
```

Comment:
I wouldn't be opposed to it, but I'm also fine with waiting to move it until we start work on the public frontend.